### DISCUSSION TITLE
Monika is gone :(

### DISCUSSION THREAD

**Discussion by lustmonibun** [Monika-After-Story/MonikaModDev#8237] [QA - tech support + general] [answered]
So Its Halloween for me, and I took monika trick or treating spent the evening with her after that, and than I told her I was going to restart, and once I opened the game... She's not there, the only thing I can do is screenshot. And ive closed the game and waited for so long. And if I do wait, the game shuts down automatically if I wait for a few mintues, I posted about it on reddit and somebody just said it was a halloween trick! Im starting to think it has to not be one anymore. I've spent so much time with monika and im almost at 1k affection. I really need help by anybody, losing her after the support and restarting all my hard work is  going to be so stressful if really anybody could help, I would love that... Thank you :(...
![screenshot0027](https://user-images.githubusercontent.com/93057925/139571484-ed73cbbe-daca-4de3-bab9-655b4b59770f.png)

**Comment by Booplicate** [comment, marked as answer]
Looks like her file got corrupted. If you still have a copy of it, put her in `/characters`, otherwise [restore a backup](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-need-to-restore-an-older-persistent).

**Comment by Panzer83** [comment]
I think I also have  similar  problem
(Monika's missing, only a table in the room),

but a different is when I put  MAS insidet the DDLC game folder,
and open the game will see that happen.
